app/src/main/java/com/github/polypoly/app/menu/SettingsActivity.kt

```diff
@@ -31,6 +33,7 @@ class SettingsActivity : MenuActivity("Settings") {
         setContent {
             MenuContent {
                 SettingsContent()
+                SignOutButton()
```

Comment:
You should put `SignOutButton()` in the code of `SettingsContent`, `MenuContent` is made to contain only a global @Composable, even if this works, it's better to stick to the mentionned convention :)

Reply:
done, thanks for the tip :)!